portfolio/src/main/webapp/script.js

```diff
@@ -29,11 +29,20 @@ function addRandomGreeting() {
 
 function addMessage() {
   fetch('/data').then(response => response.json()).then((message) => {
-    const messageList = document.getElementById('message-container')
+    
+    const messageList = document.getElementById('past-messages');
     messageList.innerHTML = '';
-    console.log(message)
+    console.log(message[0]);
     for (var i =0; i < message.length; i++){ 
```

Comment:
Should this loop stop at `message.length - 1` so it doesn't append the current message, which is added below?